uber/configspec.ini

```diff
@@ -91,12 +91,18 @@ untransferable_attrs = string_list(default=list('first_name','last_name','email'
 # a list of department constants you want to be excluded from the shift system.
 shiftless_depts = string_list(default=list())
 
```

Comment:
I figured it was worth making this configurable, even if it probably won't change for any MAGFest-run event.